tests/testthat/test-find_dependencies.R

```diff
@@ -0,0 +1,41 @@
+test_that("outputs the expected tibble", {
```

Comment:
@jacobvjk, play with this test code to try figure out how it works. 
You can run it interactively and also via `devtools::test()` (or control + shift + T)